src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -1417,6 +1427,13 @@ void scrollElementToCenterAndClick(WebElement element) {
         element.click();
     }
 
+    /**
+     * Scrolls element to first cell in spreadsheet and clicks on it.
+     */
+    void scrollElementToFirstCellAndClick(WebElement element) {
+        new Actions(browser.driver).moveToElement(element, 60, 35).click().perform();
```

Comment:
Doesn't seem like a good idea to include magic numbers.

However, the coordinates of the pixel values are generated by `handsontable` itself when it is loaded (i.e. I can't access it before it is generated). I got the values from I was inspecting element in the developer tools.
![image](https://user-images.githubusercontent.com/25320782/40634640-9b67b928-6328-11e8-91da-c23663e69669.png)

Any better suggestions?

Reply:
I think  you can look into finding the element of the 1st cell by xpath.